### ISSUE TITLE
sega multi emulators same core

### ISSUE THREAD

**Issue by joonboi** [thebezelproject/BezelProject#97]
is there any sort of workaround to the sega emulation issue. you have 4-5 emulators sharing the same cores, and some games are exactly the same name on each (sonic the hedgehog came out on SMS, Genesis, AND Game Gear) is there ever going to be a workaround for this so that my SMS version of Sonic has its own bezel, as well as my GG and Genesis ones? because in all reality the best cores for these systems will still have the same issue as there aren't enough cores to run one system per core. (And for 32x and 32XCD there's only ONE option) having the bezel overwritten with one from a different system breaks the professional look of my setup

**Comment by joonboi** [comment]
didnt realise i already mentioned something back in february...  didn't mean to make a  duplicate issue

**Comment by thebezelproject** [comment]
Yes, there is a work-around and your indicated comment above has been worked on for a period of time. A new build of the Windows bezel installation app will address this very thing.
